src/utils/liker-land-app.ts

```diff
@@ -17,13 +17,27 @@ import {
 
 import { convertWalletConnectAccountResponse } from './wallet';
 
+// Ref: https://github.com/likecoin/likecoin-app/blob/b1109871821b20228bf54cf736c032a8e9fe6ed0/app/services/api/api-config.ts#L6-L7
+export const checkIsInLikerLandAppInAppBrowser = () =>
+  navigator.userAgent.includes('LikeCoinApp');
+
 export function getLikerLandAppWCConnector(
   options: Partial<IWalletConnectOptions> = {}
 ) {
-  return new WalletConnect({
+  const wc = new WalletConnect({
     signingMethods: ['cosmos_getAccounts', 'cosmos_signAmino'],
     ...options,
   });
+
+  if (checkIsInLikerLandAppInAppBrowser()) {
+    // Ref: https://github.com/osmosis-labs/osmosis-frontend/blob/49bede85f9a772fc40ffcdcd03d193b4d8178179/packages/web/hooks/use-keplr/context.tsx#L133
+    // @ts-ignore
+    wc._clientMeta = {
+      name: 'Liker Land App (in-app)',
+    };
```

Comment:
What does this do?

Reply:
Override liker.land default meta to Liker Land app

Reply:
On second thought, I decided to reuse the user agent provided by the app as the client name